src/DIRAC/Resources/Computing/ARCComputingElement.py

```diff
@@ -32,11 +32,16 @@
 
      (key = "value")
 
+   Please note that for ARC & ARC6, any times (such as wall or CPU time) in the XRSL should be specified in minutes.
+   For AREX, these times should be given in seconds.
```

Comment:
You might want to add a link pointing to the official documentation even if it is not crystal clear:


```suggestion
   For AREX, these times should be given in seconds (see https://www.nordugrid.org/arc/arc6/users/xrsl.html?#cputime).

```